shared/middleware/helpers/property_bag.rb

```diff
@@ -56,3 +56,95 @@ def to_hash()
   end
 
 end
+
+class DynamoPropertyBag
+
+  class NotFound < Sinatra::NotFound
+  end
+
+  def initialize(channel_id, storage_id)
+    channel_owner, @channel_id = storage_decrypt_channel_id(channel_id)
```

Comment:
Useless assignment to variable - `channel_owner`. Use `_` or `_channel_owner` as a variable name to indicate that it won't be used.